server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/ContestDownloadByID.java

```diff
@@ -25,7 +28,7 @@
  * @version 0.0.1
  */
 @SuppressWarnings("PMD.AtLeastOneConstructor")
-public class ContestDownloadByID extends AbstractEndpoint {
+public class ContestDownloadByID extends AbstractCountyDashboardEndpoint {
```

Comment:
Indeed.  There were a couple of these that I couldn't tell which dashboard they were for.  Given that they are internal private skip events, it doesn't really matter.  In fact, maybe this kind of situation should just inherit from `AbstractEndpoint`?